modules/ROOT/pages/merge-types.adoc

```diff
@@ -103,13 +104,12 @@ country:
 
 == Merge Two Types When Only One Is Collaboration Enabled
 
-You can also merge two types when only one of the types has collaboration enabled. In this case, the merging causes your current type (the type you are configuring to merge) to act as a reference to the target type (the type you are merging with). Note that for the merge to happen, the target type must have collaboration enabled, and both types must define a primary key.
+You can also merge two types when only one of the types has collaboration enabled. In this case, the merging causes your current type (the type you are configuring to merge) to act as a _reference_ to the target type (the type you are merging with). Note that for the merge to happen, the target type must have collaboration enabled, and both types must define one primary key or use xref:merging-types-that-use-composite-keys[composite keys].
```

Comment:
"...and both types must define..." : Does this mean that both types must define one or the other query method, or that both types must define the same query method?